core-bundle/src/EventListener/RobotsTxtListener.php

```diff
@@ -39,22 +40,26 @@ public function __invoke(RobotsTxtEvent $event): void
         $this->contaoFramework->initialize();
 
         $file = $event->getFile();
-        $inspector = new Inspector($file);
+        $inspector = new Inspector($file, '*');
```

Comment:
_Note:_ I know this isn't necessary, but I think it makes it makes it more clear, that we are inspecting for the `*` User-Agent, as we are adding it later on, if no `user-agent` directive is present.

Reply:
Yeah I remember thinking about doing that too :) I agree, we should add it.